### DISCUSSION TITLE
Upgrading Github-Action to Leverage oasdiff-service

### DISCUSSION THREAD

**Discussion by effoeffi** [Tufin/oasdiff#453] [General]
👋 We're planning a significant enhancement for our [oasdiff-action](https://github.com/oasdiff/oasdiff-action), and we'd love your input.

Currently, our [oasdiff-action](https://github.com/oasdiff/oasdiff-action) involves 2 redundant calls to the oasdiff functionality, leading to inefficiencies. To streamline this, we aim to transition to the oasdiff-service, consolidating these calls into one for:
1. Generating github-action step output.
2. github-action annotation format output.
3. Adding Changelog and badge generation. 

This upgrade is enabler for new features, including a hosted changelog and a version-badge with color indicators. Embrace efficiency and enhanced visibility with this optimized workflow 🚀

**Comment by bdoughan-snow** [comment]
Could annotation output be added to the OpenAPI document itself like the Spectral GitHub action does?
![InlineAnnotations](https://github.com/Tufin/oasdiff/assets/65614431/cb7cdaae-a108-49f1-8416-2f64d54f16f8)
